pkg/inst/tests/test_jsonFile.R

```diff
@@ -0,0 +1,16 @@
+library(testthat)
+
+context("the jsonFile() function")
+
+sc <- sparkR.init()
+
+sqlctx <- sparkRSQL.init(sc)
+
+jsonPath <- paste(getwd(), "/pkg/inst/tests/people.json", sep = "")
+
+test_that("jsonFile() on a local file returns a DataFrame", {
+  df <- jsonFile(sqlctx, jsonPath)
+  expect_true(inherits(df, "DataFrame"))
+  expect_true(count(df) > 0)
```

Comment:
@concretevitamin Can you clarify?  Do you mean drop the `inherits` test on line 13?